docs/get-started/best-practices.md

```diff
@@ -0,0 +1,113 @@
+nav_order: 15
+parent: Getting-Started
+---
+
+# Best Practices on Spark RAPIDS
+
+This article explains the most common best practices using the RAPIDS Accelerator especially for 
+performance tuning and troubleshooting.
+
+## Workload Qualification
+
+By following [Workload Qualification](./getting-started-workload-qualification.md), you can identify
+the best candidate Spark applications for the RAPIDS Accelerator and also the feature gaps.
+
+Based on [Qualification tool](../spark-qualification-tool.md)'s output, you can start with the top N
+recommended CPU Spark jobs, especially if those jobs are computation heavy jobs (Large Joins, 
+HashAggregate, Windowing, Sorting).
+
+After those candidate jobs are run on GPU using the RAPIDS Accelerator, check the Spark driver 
+log to find the not-supported messages starting with `!`. Some missing features might be enabled by
+turning on some config, but please understand the corresponding limitation detailed in 
+[the advanced configuration document](../additional-functionality/advanced_configs.md).
+For other unsupported features, please file feature request on 
+[spark-rapids github repo](https://github.com/NVIDIA/spark-rapids/issues) with a minimum reproduce
+and the not-supported messages if they are non-senstive data.
+
+## Performance Tuning
+Refer to [Tuning Guide](../tuning-guide.md) for more details.
+
+## How to handle GPU OOM issues
+
+Spark jobs running out of GPU memory is the most common issue because GPU memory is usually much 
+smaller than host memory. Below are some common tips to help avoid GPU OOM issue.
+
+### 1. Reduce the number of concurrent tasks per GPU
+
+This is controlled by `spark.rapids.sql.concurrentGpuTasks`. Try to decrease this value to 1. 
+If the job still hit OOMs, try the following steps.
+
+### 2. Install CUDA 11.5 or above version
+
+The default value for `spark.rapids.memory.gpu.pool` is changed to `ASYNC` from `ARENA` for CUDA 
+11.5+. For CUDA 11.4 and older, it will fall back to `ARENA`.
+Using ASYNC allocator can avoid some memory fragmentation issues.
+
+The Spark executor log will contain a message like the following when using the ASYNC allocator:
+
+```
+INFO GpuDeviceManager: Initializing RMM ASYNC pool size = 17840.349609375 MB on gpuId 0
+```
+
+### 3. Identify which SQL, job and stage is involved in the error
+
+The relationship between SQL/job/stage are: Stage belongs to a Job which belongs to SQL.
+Firstly check Spark UI to identify the problematic SQL ID, Job ID, and Stage ID.
+
+Then find the failed stage in the `stages` page in Spark UI, and go into that stage to look at tasks.
+If some tasks completed successfully while some tasks failed with OOM, check the amount of input 
+types or shuffle bytes read per task to see if there is any data skew.
+
+Check the DAG of the problematic stage to see if there are any suspicious operators which may 
+consume huge amounts of memory, such as windowing, collect_list/collect_set, explode, etc. 
```

Comment:
GPUExpend is another more commonly used Expression, shall we also add it here?

Reply:
Added expand